### DISCUSSION TITLE
Docker images tags

### DISCUSSION THREAD

**Discussion by radiusmk** [Secure-Compliance-Solutions-LLC/GVM-Docker#314] [Q&A]
Where can I found information about diferences between docker images tags? A saw in issue 312 these tags:

docker pull securecompliance/gvm:debian-master-data-full
docker pull securecompliance/gvm:debian-master-data
docker pull securecompliance/gvm:debian-master-full
docker pull securecompliance/gvm:debian-master

I'm trying to install GVM and Scanner using documentation https://securecompliance.gitbook.io/projects/installing_docker, but in reports scanners show "Report outdated / end-of-life Scan Engine" for each verified host, so, I want to get the latest images versions.

Thanks.

Marco Aurélio

**Comment by austinsonger** [comment]
```
docker pull securecompliance/gvm:debian-master-data-full
docker pull securecompliance/gvm:debian-master-data
docker pull securecompliance/gvm:debian-master-full
docker pull securecompliance/gvm:debian-master
```
Those are the latest image versions, they have been tested. 

Updating the OpenVAS Docker Image this weekend.

If you are running into any issues please created a new issue and follow the directions.

**Reply by radiusmk** [reply]
But what is the diference between each image? Are there same tags for Openvas?

Tks for reply.

**Reply by devMarc5598** [reply]
Do you know the difference now?

**Comment by Dexus** [comment]
You can now use the https://hub.docker.com/r/deineagenturug/gvm images, which are using the latest gvmd and components releases.

It use POSTGRESQL 13. 

What are meaning the tags:

```
docker pull deineagenturug/gvm:latest               # is an not initialized basic image, that first start its initialization on the first start (short: no db & no PDF report support)
docker pull deineagenturug/gvm:latest-full          # is an not initialized basic image, that first start its initialization on the first start (short: no db, PDF report is support)
docker pull deineagenturug/gvm:latest-data          # is an initialized basic image, that can run directly without extra VOLUMES added (short: included db, but no PDF report support)
docker pull deineagenturug/gvm:latest-data-full     # is an initialized basic image, that can run directly without extra VOLUMES added (short: included db & PDF report is support)
```

It can be used as direct replacement for the here provided image, with the same arguments on `docker run`.
